src/main/java/frc/robot/subsystems/vision/VisionSubsystem.java

```diff
@@ -147,44 +158,25 @@ public PoseEstimate getMegaTag2PoseEstimate(int limelightNumber) {
     return LimelightHelpers.getBotPoseEstimate_wpiBlue_MegaTag2(getLimelightName(limelightNumber));
   }
 
-  /**
-   * Checks if the MegaTag1 and MegaTag2 pose estimates are within the field parameters
-   *
-   * @param limelightNumber the number of the limelight
-   * @return true if the poses are within the field, false if not.
-   */
   public boolean isValidPoseEstimate(int limelightNumber) {
-    return isMegaTag1Good(limelightNumber) && isMegaTag2Good(limelightNumber);
-  }
-
-  /**
-   * Checks if the MegaTag1 pose estimate is within the field parameters
-   *
-   * @param limelightNumber the number of the limelight
-   * @return true if the pose is within the field, false if not.
-   */
-  public boolean isMegaTag1Good(int limelightNumber) {
     PoseEstimate megaTag1Estimate = getMegaTag1PoseEstimate(limelightNumber);
+    PoseEstimate megaTag2Estimate = getMegaTag2PoseEstimate(limelightNumber);
 
-    return ((megaTag1Estimate.pose.getX() > 0
-            && megaTag1Estimate.pose.getX() <= FieldConstants.FIELD_WIDTH_METERS)
-        && (megaTag1Estimate.pose.getY() > 0
-            && megaTag1Estimate.pose.getY() <= FieldConstants.FIELD_WIDTH_METERS));
+    return LimelightHelpers.isValidPoseEstimate(megaTag1Estimate)
+        && LimelightHelpers.isValidPoseEstimate(megaTag2Estimate)
+        && isWithinFieldBounds(megaTag1Estimate, megaTag2Estimate);
   }
 
-  /**
-   * Checks if the MegaTag2 pose estimate is within the field parameters
-   *
-   * @param limelightNumber the number of the limelight
-   * @return true if the pose is within the field, false if not.
-   */
-  public boolean isMegaTag2Good(int limelightNumber) {
-    PoseEstimate megaTag2Estimate = getMegaTag2PoseEstimate(limelightNumber);
-
-    return ((megaTag2Estimate.pose.getX() > 0
+  private boolean isWithinFieldBounds(
```

Comment:
This is a good idea having this method, and I like making it private -- now that I think of it, more of the VisionSubsystem methods should prolly be private